circuits/cpp/src/aztec3/circuits/rollup/test_utils/utils.cpp

```diff
@@ -0,0 +1,374 @@
+#include "nullifier_tree_testing_harness.hpp"
+#include "utils.hpp"
+#include "aztec3/constants.hpp"
+#include "init.hpp"
+
+#include <aztec3/circuits/kernel/private/utils.hpp>
+#include <aztec3/circuits/mock/mock_kernel_circuit.hpp>
+#include "aztec3/circuits/abis/new_contract_data.hpp"
+#include "aztec3/circuits/abis/rollup/root/root_rollup_public_inputs.hpp"
+namespace {
+using NT = aztec3::utils::types::NativeTypes;
+
+using ConstantRollupData = aztec3::circuits::abis::ConstantRollupData<NT>;
+using BaseRollupInputs = aztec3::circuits::abis::BaseRollupInputs<NT>;
+using RootRollupInputs = aztec3::circuits::abis::RootRollupInputs<NT>;
+using RootRollupPublicInputs = aztec3::circuits::abis::RootRollupPublicInputs<NT>;
+using DummyComposer = aztec3::utils::DummyComposer;
+
+using Aggregator = aztec3::circuits::recursion::Aggregator;
```

Comment:
a lot of these namespaces are duplicated across init and this file

Reply:
Yes. Should be fixed, but might make sense as above?

Reply:
okay ya agreed - can do it in its own task